maturity-level-two/README.md

```diff
@@ -1,2 +1,12 @@
 # Practical API Guidelines - Should have
 
+1. [Writing Open API Specifications](#writing-open-api-specifications)
+
+## Writing solid OpenAPI Specifications
```

Comment:
I'm out of inspiration and welcome feedback for title + intro paragraph.

Reply:
Title is ok. We can add a point to the bullet list "unit test the open api document generation"

Reply:
Updated, let me know if you like it.